exporter/opentelemetry-exporter-otlp/src/opentelemetry/exporter/otlp/exporter.py

```diff
@@ -123,23 +130,36 @@ class OTLPExporterMixin(
         endpoint: OpenTelemetry Collector receiver endpoint
         credentials: ChannelCredentials object for server authentication
         metadata: Metadata to send when exporting
+        compression: Compression algorithm to be used in channel
     """
 
     def __init__(
         self,
         endpoint: str = "localhost:55680",
         credentials: ChannelCredentials = None,
         metadata: Optional[Tuple[Any]] = None,
+        compression: enum.Enum = GRPCCompression.DEFAULT,
     ):
         super().__init__()
 
         self._metadata = metadata
         self._collector_span_kwargs = None
 
+        if compression is GRPCCompression.DEFAULT:
+            compression_algorithm = Compression.NoCompression
+        elif compression is GRPCCompression.GZIP:
```

Comment:
How about using `if gzip else` - I'm guessing `None` is valid for passing to `insecure_channel` but it doesn't hurt to be explicit with our default.